src/pip/_internal/commands/configuration.py

```diff
@@ -237,13 +241,32 @@ def open_in_editor(self, options, args):
             raise PipError("Could not determine appropriate file.")
 
         try:
-            subprocess.check_call([editor, fname])
+            if PY3:
```

Comment:
You can also note in a comment that this check won't be needed once we drop Python 2 support

Reply:
I think that's self-explanatory. :)

Reply:
Hi! :-)

Should this conditional be inverted, and made to use `if PY2:` instead? (With the `args = self._get_editor_cmd(editor)` in the `else` clause.)

That way it will be compatible with future Python 4+. (Given that pip released this year will still likely be in use in some environments for years to come.)

Reply:
> future Python 4+.

We can work on that when it's a possibility that CPython releases a Python 4 -- currently, there are no plans for it AFAIK, so future proofing for that is... not super useful IMO.